### DISCUSSION TITLE
Account creation success page not redirecting properly

### DISCUSSION THREAD

**Discussion by masterboy9** [hrfee/jfa-go#170] [Q&A]
My setup: jfa-go set up on accounts.mydomain.com, while my jellyfin server is on mydomain.com. In the jfa-go settings, the server is set up as mydomain.com (My reverse proxy set up takes accounts.mydomain.com to jfa-go, and mydomain.com to jellyfin)

When a user makes an account on jfa-go and gets the success message, the account is created and they can login perfectly fine. However, when they click the continue button on that success screen after the account creation, it redirects them to accounts.domain.com/invite/mydomain.com instead of just mydomain.com. This has the effect of redirecting my users to the invalid invite code page after they already successfully made an account. I'm not sure what is causing this, and would love some help.

**Comment by hrfee** [comment]
Sorry for such a late reply, looking through the code the value is filled in from the "Public Server" option in Settings > Jellyfin. This has been the case since at least 0.3.9 but I may have accidentally changed it in a previous version then fixed it again. Could you check which version you're running?

**Reply by 886ppak** [reply]
morning hrfree,

Im having the exact problem as above and im running on docker using the lastest pull version as of 11/5/22
any help would be really appeciated in solving this issue 

kinds regards
aiden

**Reply by 886ppak** [reply]
disregard i got it to work :D 👍 👍
